src/Infrastructure/PayPalPaymentNotificationVerifier.php

```diff
@@ -56,14 +65,23 @@ public function verify( array $request ) {
 		}
 	}
 
-	private function matchesReceiverAddress( $request ): bool {
+	private function matchesReceiverAddress( array $request ): bool {
 		return array_key_exists( 'receiver_email', $request ) &&
 			$request['receiver_email'] === $this->config['account-address'];
 	}
 
-	private function isPaymentStatusAllowed( $request ): bool {
+	private function isPaymentStatusAllowed( array $request ): bool {
```

Comment:
bit weird wording: I'd go for something like `paymentStatusIsAllowed` or `isAllowedPaymentStatus`